terraform/main.mk

```diff
@@ -87,9 +87,9 @@ terraform.13upgrade:
 	@ $(CMD_TERRAFORM_MODULES_UPGRADE)
 
 terraform.apply: terraform.plan ## Deploy infrastructure
-	@ cd $(ENV_DIR) && \
-	$(TERRAFORM) apply -input=false tfplan && \
-	$(TERRAFORM) output -json > output.json	&& \
+	@ cd $(ENV_DIR)/.terraform && \
```

Comment:
Please explain why are we changing terraform root to .terraform, can't we point to the file without cd-ing into the .terraform?

Reply:
We can. Fixed this, thank you.